lib/matplotlib/axes/_axes.py

```diff
@@ -1978,7 +1978,9 @@ def acorr(self, x, **kwargs):
 
         Parameters
         ----------
-        x : array-like
+        x : array-like.
```

Comment:
The type doesn't need to be a sentence.
```suggestion
        x : array-like
```